nightwatch_tests/smoke_tests.js

```diff
@@ -24,55 +24,28 @@ module.exports = {
 
   "4. Mood images are present": function (browser) {
     browser.url("https://project32.s3.eu-west-2.amazonaws.com/index.html");
-    
+
     const cssSelector = "#root > div > div > div > div";
     browser.expect.element("#root > div > div > div > div").to.be.visible;
```

Comment:
Avoid repeating yourself - you've defined the selector string, so make use of it! This also applies at line 8, 15 and 36.
```suggestion
    browser.expect.element(cssSelector).to.be.visible;
```